flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/MeasurementInfoProviderTest.java

```diff
@@ -33,16 +38,33 @@
 import static org.junit.Assert.assertThat;
 
 /** Test for {@link MeasurementInfoProvider}. */
+@RunWith(Parameterized.class)
 public class MeasurementInfoProviderTest extends TestLogger {
+
+    private static final String KEY1 = "A";
+    private static final String KEY2 = "B";
+    private static final String KEY3 = "C";
+    private static final String VALUE1 = "a";
+    private static final String VALUE2 = "b";
+    private static final String VALUE3 = "c";
+
     private final MeasurementInfoProvider provider = new MeasurementInfoProvider();
 
+    private final boolean isDelimited;
+
+    @Parameters(name = "isDelimited = {0}")
+    public static List<Boolean> parameters() {
+        return Arrays.asList(false, true);
+    }
+
+    public MeasurementInfoProviderTest(boolean isDelimited) {
+        this.isDelimited = isDelimited;
+    }
+
     @Test
     public void simpleTestGetMetricInfo() {
         String logicalScope = "myService.Status.JVM.ClassLoader";
-        Map<String, String> variables = new HashMap<>();
-        variables.put("<A>", "a");
-        variables.put("<B>", "b");
-        variables.put("<C>", "c");
```

Comment:
please add a _dedicated_ test for the removal of newline characters in variables. This test already does more than enough.

Reply:
ok